jbpm-container-test/jbpm-remote-ejb-test/jbpm-remote-ejb-test-app/src/main/resources/META-INF/persistence.xml

```diff
@@ -68,6 +68,7 @@ http://java.sun.com/xml/ns/persistence/orm http://java.sun.com/xml/ns/persistenc
 			<!-- BZ 841786: AS7/EAP 6/Hib 4 uses new (sequence) generators which seem 
 				to cause problems -->
 			<property name="hibernate.id.new_generator_mappings" value="false" />
+			<property name="hibernate.enable_lazy_load_no_trans" value="true" />
```

Comment:
Isn't it even a bug in remote EJBs? Why do we have to "hack" this externally with the Hibernate property?